specification/securityinsights/resource-manager/common/1.0/types.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Please do not add ARM common definition here. Please reference /specifications/common-types/resource-manager/[V1/V2. V2 recommended]/types.json